doc/xbgpu.design.b.rst

```diff
@@ -86,3 +86,35 @@ scaling to convert to a real value in :math:`(-0.5, 0.5)`. While this is
 still a deviation from uniformity, it does give a symmetric distribution.
 
 .. _curand: https://docs.nvidia.com/cuda/curand/index.html
+
+Data flow
+---------
+The host side of the beamforming is simpler than for correlation, because
+there is no accumulation. For simplicity, the output chunk size (in time) is
```

Comment:
I haven't reviewed any of your documentation in a while. I see you've used commas in what I would call "an interesting way".

Of course, my this is my opinion and not a hill I'm willing to die on. Still, might I suggest the first sentence simply reads without the comma. I don't feel a need to pause before reading the reasoning portion of the sentence.
> The host side of the beamforming is simpler than for correlation because there is no accumulation.